cylc/flow/platforms.py

```diff
@@ -652,7 +654,7 @@ def get_install_target_to_platforms_map(
     Return {install_target_1: [platform_1_dict, platform_2_dict, ...], ...}
     """
     ret: Dict[str, List[Dict[str, Any]]] = {}
-    for p_name in set(platform_names) - set(JOBLESS_MODES):
+    for p_name in set(platform_names):
```

Comment:
The jobless logic here is not needed with my change to deep copy the localhost platform above